### PR TITLE
Can vote on status

### PR DESCRIPTION
**Pull request by valentin-helies**

### CODE REVIEW

File: src/Resources/views/credential/index.html.twig [hunk 1]
```diff
@@ -64,12 +64,65 @@
                     </tr>
             {% endif %}
                     <tr class="rubrique-groupe-data-{{ rubriqueId }}">
-                        <th>{{ cred.libelle ?? cred|lower|humanize }}</th>
+                        <th>
+                            {{ cred.libelle ?? cred|lower|humanize }}
+                            {% if cred.listeStatus %}
+                                <hr>
+                                <div>
+                                    Allowed Status
```
Comment (SpaghettiBolognaise) [inline src/Resources/views/credential/index.html.twig]: traductions

File: src/Security/CredentialVoter.php [hunk 2]
```diff
@@ -61,8 +61,8 @@ public function __construct(AccessDecisionManagerInterface $decisionManager, Ent
 
     protected function supports($attribute, $subject): bool
     {
-         // vote on everything
-         if (!in_array($attribute, ['IS_AUTHENTICATED_REMEMBERED','ROLE_USER','IS_AUTHENTICATED_ANONYMOUS','IS_AUTHENTICATED_FULLY','ROLE_SUPER_ADMIN'])) {
```
Comment (SpaghettiBolognaise) [inline src/Security/CredentialVoter.php:65]: pourquoi enlever ROLE_SUPER_ADMIN ?

File: src/Resources/translations/CredentialBundle.fr.yaml [hunk 3]
```diff
@@ -3,6 +3,7 @@ title:
     rights: "Droits"
     unknown_heading: "Rubrique inconnue"
     all_roles: "Tous les rôles"
+    status_allowed: "Autoriser par status"
```
Comment (SpaghettiBolognaise) [inline src/Resources/translations/CredentialBundle.fr.yaml]: "statut" in french

File: src/Security/CredentialVoter.php [hunk 4]
```diff
@@ -12,6 +14,7 @@
 use Symfony\Component\Cache\Adapter\AdapterInterface;
 use Symfony\Component\Security\Core\Authorization\AccessDecisionManagerInterface;
 use Psr\Cache\CacheItemPoolInterface;
+use function Composer\Autoload\includeFile;
```
Comment (SpaghettiBolognaise) [inline src/Security/CredentialVoter.php]: w0t

**Review by SpaghettiBolognaise — APPROVED** [review]

File: src/Security/CredentialVoter.php [hunk 5]
```diff
@@ -99,14 +119,43 @@ protected function voteOnAttribute($attribute, $subject, TokenInterface $token):
         if (in_array('ROLE_SUPER_ADMIN', $roles)) {
             return true;
         }
-        foreach($roles as $role) {
-            $k = str_replace('ROLE_', '', $role);
-            if (isset($this->groupRights[$k]) && in_array($attribute, $this->groupRights[$k])) {
+
+        $roleWithStatus = $this->em->getRepository(Credential::class)->findOneBy(["role" => $attribute]);
+
+        foreach ($roles as $role) {
```
Comment (sebheitzmann) [inline src/Security/CredentialVoter.php:128]: non non non

**Pull request merged**